### ISSUE TITLE
New install won't run the markdown unless .styleguide file is renamed to .styleguide.json but with errors

### ISSUE THREAD

**Issue by stantonl33** [UWHealth/markdown-documentation-generator#22]
So I added the generator to my project and added the markdown to my SASS files. When I tried running the `md_documentation` command it would give me the unexpected token ":" error. For the heck of it I renamed the .styleguide file to .styleguide.json and it ran. 

The html seems fine but it doesn't show the previews of the code examples.

At the end of it running it threw this error:
`(node:12996) UnhandledPromiseRejectionWarning: TypeError: callback is not a function
    at C:\Users\...\AppData\Roaming\npm\node_modules\markdown-documentation-generator\index.js:776:29
(Use 'node --trace-warnings ...' to show where the warning was created)
(node:12996) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag '--unhandled-rejections=strict' (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:12996) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`

**Comment by luigimannoni** [comment]
Hey, which node version are you running? I've seen this happening on node v15 and above.

Also, you can try this fork and let me know if it works for you https://www.npmjs.com/package/@deimosindustries/markdown-documentation-generator

**Comment by stantonl33** [comment]
I'm running node version 14.17.0 at the moment. 

**Side note:** I managed to get the html previews to work. Had to add the "html_example" words after the first set of 3 tildes. ```html_example

This wasn't required in the initial version of this package by emiloberg, that I know of.

So as of right now I have this fork working for me except the javascript "toggles" do nothing for some reason at this time. So I have to figure that out yet. In time I may be able to figure out why. We'll see when I can get to it.

In summary, I would say having to rename the .styleguide file to .styleguide.json as well as the Javascript toggles in the generated styleguide not working are open issues with this fork currently.

**Comment by luigimannoni** [comment]
Thanks for trying it out.
Sorry, should've said that I've kept the `.json` extension for config files as node was desperately trying to interpret a json file without extension as such. I've changed the method to retrieve the config file and now the `.styleguide` file is working again.

Regarding the ` ```html_example ` identifier I can't really help with that, since I've started to use this tool I've always used the html_example sections. Perhaps if you keep the config key empty like such `"exampleIdentifier": ""` it would probably start working again as you expect to be?

Anyways, I've published an updated version with some of the fixes above, you can try it again running `npm install @deimosindustries/markdown-documentation-generator@^3.2.5`

I've tested it with both node 14.17 and node 15 and it should work as expected.

If you have any other issues, please raise them in the forked repository here https://github.com/deimosindustries/markdown-documentation-generator so I can keep track of them better.
